pulsar-client/src/main/java/org/apache/pulsar/client/api/Message.java

```diff
@@ -62,6 +63,8 @@
      */
     byte[] getData();
 
+    T getEvent();
```

Comment:
I don't have a strong preference here, though maybe consider `getValue()`

Reply:
renamed to getValue